ci/update-checker/src/main/scala/com/twitter/intellij/updatechecker/UpdateChecker.scala

```diff
@@ -80,14 +89,32 @@ object UpdateChecker {
       case p: Versioned => findUpdates(p)
     }.flatten
 
+    val hasPlatformUpdate = {
+      import buildOrdering._
+      platformUpdate > config.intellij
+    }
+
+    if (hasPlatformUpdate) {
+      println(s"A platform update is available: ${platformUpdate.build}")
+    } else {
+      println("The platform is up to date!")
```

Comment:
Can we just call it 'intellij' rather than 'platform'? Same as we refer to it when we talk about updates (updating intellij and plugins).

Reply:
Modified the message